images/macos/tests/Xamarin.Tests.ps1

```diff
@@ -180,6 +186,25 @@ Describe "Xamarin.Android" {
                 Join-Path $xamarinLibPath "xbuild-frameworks" | Should -Exist
             }   
         }
+    }
+
+        Context "NDK toolchains" -Skip:($os.IsBigSur) {

```

Comment:
I think it should be separate describe or even separate file because Xamarin NDKs are installed in the separate test file: `xamarin-android-ndk.sh` that is run separately from `xamarin.sh`

Reply:
Thank for the good point, moved it to separate file.